pkg/userdata/ubuntu/testdata/dist-upgrade-on-boot.yaml

```diff
@@ -156,6 +157,7 @@ write_files:
       socat \
       util-linux \
       ${CR_PKG} \
+      ${CR_CLI_PKG} \
       ipvsadm
     apt-mark hold docker.io || true
```

Comment:
I think this line is there for historical reasons and could be removed, isn't it?

Reply:
We have agreed to keep it for now and get rid of it in a follow up.